lib/modules/hosts/wikipedia.js

```diff
@@ -55,6 +55,8 @@ export default {
 		$wikiData.find('.metadata, .hatnote, .mw-editsection, .reference, .references').remove();
 		// Make all relative links direct
 		$wikiData.find('a[href^="/"]').attr('href', (i, old) => `https://${language}.wikipedia.org${old}`);
+		// Prepend wikipedia logo to make clear it is a wikipedia article
+		$wikiData.prepend('<img src="https://en.wikipedia.org/static/favicon/wikipedia.ico" alt="Hosted on wikipedia" title="Hosted on wikipedia">')
```

Comment:
Note that this would only affect expandos with `type: 'TEXT`: 
- Deviantart (https://www.reddit.com/r/twitchplayspokemon/comments/53jkmg/kakuna_wars_chapter_six_phone_home/)
- Tumblr (https://www.reddit.com/r/ShingekiNoKyojin/comments/53rhnt/manga_spoilers_theory_titans_true_objectives/)
- Github (https://www.reddit.com/r/privacytoolsIO/comments/53hjkt/howto_privacy_security_conscious_browsing/)